packages/git/src/node/git-backend-module.ts

```diff
@@ -70,15 +72,18 @@ export function bindGit(bind: interfaces.Bind, bindingOptions: GitBindingOptions
     bind(CommitDetailsParser).toSelf().inSingletonScope();
     bind(GitBlameParser).toSelf().inSingletonScope();
     bind(GitExecProvider).toSelf().inSingletonScope();
+    bind(DugiteGit).toSelf().inSingletonScope();
+    bind(Git).toService(DugiteGit);
+    bind(DefaultGitInit).toSelf();
+    bind(GitInit).toService(DefaultGitInit);
```

Comment:
I still don't get why this is actually working :-( 
I'd expected this to fail as `DefaultGitInit` depends on `MessageService` bound in connection scope.
any idea why?
@kittaakos, this used to fail, no?